apps/laboratory/tests/smart-account.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { testModalSmartAccount } from './shared/fixtures/w3m-smart-account-fixture'
+import type { ModalWalletPage } from './shared/pages/ModalWalletPage'
+import type { ModalWalletValidator } from './shared/validators/ModalWalletValidator'
+
+testModalSmartAccount.beforeEach(async ({ modalValidator }) => {
+  await modalValidator.expectConnected()
+})
+
+testModalSmartAccount('it should sign with eoa', async ({ modalPage, modalValidator }) => {
+  await modalPage.sign()
+  await modalPage.approveSign()
+  await modalValidator.expectAcceptedSign()
+})
+
+testModalSmartAccount(
+  'switch to its smart account',
```

Comment:
Let's keep the testing naming convention:
```suggestion
  'it should switch to its smart account',
```